code/ATMOSPHERICS/components/unary_devices/outlet_injector.dm

```diff
@@ -177,6 +232,14 @@
 "}
 
 /obj/machinery/atmospherics/unary/outlet_injector/attackby(obj/item/W, mob/user)
+	if(istype(W, /obj/item/pen))
+		var/t = copytext(stripped_input(user, "Enter the name for the injector.", "Rename", name), 1, MAX_NAME_LEN)
```

Comment:
why do you use copytext here? It can be done without since `stripped_input` already caps the character count